dev/com.ibm.ws.kernel.feature.core/src/com/ibm/ws/kernel/feature/internal/FeatureResolverImpl.java

```diff
@@ -533,6 +533,9 @@ private boolean isAllowedToleration(SelectionContext selectionContext, Provision
         if (overrideTolerates.contains(tolerate)) {
             return true;
         }
+        if(!chain.peekFirst().contains("-")){
```

Comment:
I think this needs to be scoped down a bit more.  I'm not convinced all product extensions or user features follow this convention of having `-<version>` at the end of their feature.

Instead I think we should depend on a prefix like `io.openliberty.versionless.` or `com.ibm.websphere.versionless.`